app/forms/registration_form.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class RegistrationForm < ApplicationForm
+  attr_accessor :email, :password, :password_confirmation, :client_id
```

Comment:
```suggestion
```

We don't need this definition right? 🤔

Reply:
Currently it was needed for the `initialize` method that acts as a Model.

But as Form objects must be initialized with the models (or no attributes) and must call `save` with the params, they won't be needed.